packages/features/shell/Shell.tsx

```diff
@@ -1053,7 +1053,7 @@ export function ShellMain(props: LayoutProps) {
         </div>
       )}
       {props.afterHeading && <>{props.afterHeading}</>}
-      <div className={classNames(props.flexChildrenContainer && "flex flex-1 flex-col")}>
+      <div className={`${classNames(props.flexChildrenContainer && "flex flex-1 flex-col")} px-2`}>
```

Comment:
same here, no need to add px-2 if you can add it inside flex, flex-1 and flex-col